product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/third_party_integrations/datadog.mdx

```diff
@@ -45,18 +45,22 @@ The set of metrics delivered to Datadog is subject to change. Metrics with names
 
 ## Cost
 
-After enabling the BigAnimal telemetry integration, check your billable Datadog usage and continue to monitor it over time.
+Telemetry sent to Datadog will incur charges on your Datadog plan according to the [Datadog price list](https://www.datadoghq.com/pricing/list/).
+We recommend that you continually monitor your billable Datadog usage as the cost could change.
+Datadog provides [usage metrics](https://docs.datadoghq.com/account_management/billing/usage_metrics/) for this purpose, such as `datadog.estimated_usage.hosts`, `datadog.estimated_usage.containers`, and `datadog.estimated_usage.metrics.custom`.
 
 Be aware of the following cost considerations:
 
-* *You are responsible for all costs* charged to your Datadog account by telemetry sent by the BigAnimal Datadog integration. Charges are based on usage but they can also result from BigAnimal errors or oversights. Enable the Datadog integration only if you accept this responsibility.
```

Comment:
This is mostly a solid improvement. But you've deleted this piece..

There IMO needs to be a very clear statement of responsibility about costs and cost risks. I'm not set on the wording but I think it's vital we have something.

Reply:
It felt to me that the warning was laboring the point a little. The first paragraph already states that costs will be incurred. Let me check with the BA team as to whether they think we need a more legalistic disclaimer here.

Reply:
The decision from the BA team is to stick with the proposed changes. @nidhibhammar please can you review once more and merge if ok?